source/faucet/main.d

```diff
@@ -68,7 +68,7 @@ import vibe.inet.url;
 import vibe.web.rest;
 
 /// The keys that will be used for generating transactions
-private SecretKey[PublicKey] secret_keys;
+private KeyPair[PublicKey] allKeys;
```

Comment:
```suggestion
private KeyPair[PublicKey] all_keys;
```
Is there any reason to use the name instead of `allKeys`?

Reply:
No I just copied it from the example usage in the TXBuilder documentation. I have renamed to `known_keys` which is correct style and a better name.